Library/Homebrew/cask/artifact/abstract_uninstall.rb

```diff
@@ -127,7 +127,7 @@ def running_processes(bundle_id)
           .map { |line| line.chomp.split("\t") }
           .map { |pid, state, id| [pid.to_i, state.to_i, id] }
           .select do |(pid, _, id)|
-            pid.nonzero? && id.match?(/^#{Regexp.escape(bundle_id)}($|\.\d+)/)
+            pid.nonzero? && id&.match?(/^#{Regexp.escape(bundle_id)}($|\.\d+)/)
```

Comment:
```suggestion
            pid.nonzero? && /^#{Regexp.escape(bundle_id)}($|\.\d+)/.match?(id)
```

to avoid the `nil` check.